api/api/prediction_job_api.go

```diff
@@ -103,21 +100,20 @@ func (c *PredictionJobController) Get(r *http.Request, vars map[string]string, _
 
 	model, version, err := c.getModelAndVersion(ctx, modelID, versionID)
 	if err != nil {
-		if !gorm.IsRecordNotFoundError(err) {
-			return InternalServerError(err.Error())
+		if gorm.IsRecordNotFoundError(err) {
+			return NotFound(fmt.Sprintf("Model / version not found: %v", err))
 		}
-		return NotFound(err.Error())
+		return InternalServerError(fmt.Sprintf("Error getting model / version: %v", err))
 	}
 
 	env, err := c.AppContext.EnvironmentService.GetDefaultPredictionJobEnvironment()
 	if err != nil {
-		return InternalServerError("Unable to find default environment, specify environment target for deployment")
+		return InternalServerError(fmt.Sprintf("Unable to find default environment, specify environment target for deployment: %v", err))
 	}
 
 	job, err := c.PredictionJobService.GetPredictionJob(ctx, env, model, version, id)
 	if err != nil {
```

Comment:
It's interesting how we're not performing the `if gorm.IsRecordNotFoundError(err)` check here for `PredictionJobService.GetPredictionJob` and also in other instances below, but I guess it's not super important if we don't do it, just pointing this out 🤔

Reply:
Good point. Corrected these instances.